### ISSUE TITLE
[FEATURE] More Clothes / Hair Styles

### ISSUE THREAD

**Issue by TheAlphaStrain** [iNCEPTION-RP/Feedback#29]
**Describe in as much detail as possible for the feature you are requesting.**
Pretty simple and self explanatory title as to what feature.

**In what ways would this improve the in-game experience?**
We want RP right? Let's get the clothing and hair styles for it. Why is clothing important? Okay, well, let's put it this way. There's a lot of suit & ties, hoodies pants and a bunch of what I like to call "junk" clothing that makes it look like you stepped out of Mad Max Fury Road. Want a pool party? Well, there's only like 1 shirt that I found (Yes you can take it off as male) and like 1-3 shorts that could pass as swim trunks. Or beach parties. 

Or, you wanna go to a nightclub, men got all sorts of suits but I couldn't help but notice women don't have many dresses. Or fancy hairdos. There's plenty of reasons and occasions where clothing and hair can be a good factor to RP. 

**Will this require work in other parts of the game? Example: (Inventory/Jobs)**
Probably not.

**Comment by troopakoopa** [comment]
100% agree

**Comment by ixhal** [comment]
I've said this multiple times and I don't think it was being relayed properly so I will say it publicly here for you guys.
We can NOT add anymore clothing at this time since when we try to it crashes the game because of the pool size of YMTs.. I have heard some news that this pool size will be updated on the next GTA5 dlc.. We physically cant add more clothes atm unless we replace the current ones, and that is just something we want to avoid.

**Comment by TheAlphaStrain** [comment]
I was not made aware that there was an inability to. I've not discussed this with anyone nor heard that response. I've not really cared about clothing until I started to change mine more for different things.

**Comment by llanboy** [comment]
I agree 100% but I also know that every time Al adds/removed clothing it breaks the script. I know he has been looking into it and the staff team are under no illusions as to how important custom clothing and hairstyles are to the community. I recently did a bit of research and it was overwhelming the amount of posts I seen on social media etc regarding people saying they look for custom clothing and hairstyles in a server. I'm time this will happen as Cassie I believe already has a number of clothing items ready to rock and roll 🤘

**Comment by TheAlphaStrain** [comment]
See, I didn't know any of that. Up until like the last 2-3 days I've only ever had 3 outfits saved. Now I have 7 thanks to a certain someone. They know who they are.
